kclvm/sema/src/builtin/system_module.rs

```diff
@@ -1155,7 +1155,9 @@ register_json_member! {
 // ------------------------------
 
 pub const CRYPTO: &str = "crypto";
-pub const CRYPTO_FUNCTION_NAMES: &[&str] = &["md5", "sha1", "sha224", "sha256", "sha384", "sha512"];
+pub const CRYPTO_FUNCTION_NAMES: &[&str] = &[
```

Comment:
Failure of sema/symbol related tests in API and loader.

Reply:
> Failure of sema/symbol related tests in API and loader.

Failure of sema/symbol related tests in the loader crate.

Reply:
fixed